cedar-policy-core/src/est/expr.rs

```diff
@@ -108,12 +108,6 @@ pub enum ExprNoExt {
     Var(ast::Var),
     /// Template slot
     Slot(#[cfg_attr(feature = "wasm", tsify(type = "string"))] ast::SlotId),
-    /// Unknown (for partial evaluation)
-    Unknown {
-        /// Name of the unknown
-        #[cfg_attr(feature = "wasm", tsify(type = "string"))]
-        name: SmolStr,
```

Comment:
Looking at this unknown variant closer, I see that it would accept `{ "unknown": "foo" }`, but the extension form accepts `{"unknown": [ {"value": "foo" }]}`. So, we are breaking one representation of unknown in the est. This is probably fine based on the justification that it's an experimental feature (and we're coming up on a major version anyways)

Reply:
Yes, good catch.